docs/docs-using-nodejs.md

```diff
@@ -11,15 +11,15 @@ This example shows how to use a dynamically linked Javy compiled Wasm module. We
 
 ### Steps
 
-1. The first step is to compile the `embedded.js` with Javy using dynamic linking:
+1. Emit the Javy plugin
 ```shell
-javy build -C dynamic -o embedded.wasm embedded.js
+javy emit-provider -o plugin.wasm
```

Comment:
Should this be `emit-plugin`?

Reply:
I haven't merged the PR where I rename the command yet. I assume this will register as a merge conflict if I merge that PR before this one or if I merge this PR and then try to merge that PR because PRs are changing the same line in different ways.

Reply:
And it looks like it did register as a merge conflict 🎉